### PR TITLE
Match Select Design as per Styleguide

### PR DESCRIPTION
**Pull request by azizhk**
https://gallery.io/projects/MCHbtQVoQ2HCZT4mqWZoe47O/files/MCHtA7U1iMGr6_t5bV_53j4SjSutDL__KP4

The input color is gray.darker
Wanted to leave functionality as is (i.e. readonly), so made Input accept inputClassName prop.
Avoid using `&.className` in emotion as difficult to override and requires string matching code which cannot be type checked.

### CODE REVIEW

**Comment by azizhk** [comment]
I just now needed to overrided readonly's color to support Select

File: src/components/Input.tsx [hunk 1]
```diff
@@ -75,11 +79,13 @@ class Input extends React.PureComponent<InputProps, InputState> {
...
-    const inputClassName = cx(inputStyle, {
-      _pebble_input_disabled: disabled,
-      _pebble_input_read_only: readOnly,
-      _pebble_input_textarea: textArea
-    });
+    const _inputClassName = cx(
+      inputStyle,
+      !!disabled && inputDisabledStyle,
+      !!readOnly && inputReadOnlyStyle,
+      !!textArea && inputTextAreaStyle,
```
Comment (ritz078) [inline src/components/Input.tsx]: use the recommended syntax.

```js
const _inputClassName = cx(inputStyle, {
      inputDisabledStyle : disabled,
      inputReadOnlyStyle: readOnly,
      inputTextAreaStyle: textArea
}
```

Comment (azizhk) [reply src/components/Input.tsx]: That will not work, as `inputDisabledStyle` is a variable not a string

Comment (ritz078) [reply src/components/Input.tsx]: updated

Comment (azizhk) [reply src/components/Input.tsx]: done

**Pull request merged**